Sources/WebUI/WebViewProxy.swift

```diff
@@ -186,4 +189,15 @@ public final class WebViewProxy: ObservableObject {
     public func clearAll() {
         webView?.remake()
     }
+
+    /// A container for capturing the web view's content.
+    public struct ContentReader {
+        let wkWebView: WKWebView
```

Comment:
nits

```suggestion
        var wkWebView: WKWebView
```

And in other contexts, it's named `webView` instead of `wkWebView`. What should we name this variable?

Reply:
@Kyome22 
I named it `wkWebView` to emphasize that it’s a `WebKit` `WKWebView`, but we might be able to rely on the type to make that distinction instead. Which do you think is better?

Reply:
In the overall source code, the argument names for the functions in `WKNavigationDelegate` are `webView`, and the labels for `EnhancedWKWebView` and `Remakeable<EnhancedWKWebView>` are also `webView`. Therefore, if we decide to use `wkWebView` to clarify the type, I would like to reflect that as an overall rule.

example 1

```swift
extension EnhancedWKWebView.NavigationDelegateProxy: WKNavigationDelegate {
    func webView(
        _ webView: WKWebView,
        didFail navigation: WKNavigation!,
        withError error: any Error
    ) {}
}
```

example 2

```swift
@MainActor
func applyModifiers(to webView: EnhancedWKWebView) {
    webView.uiDelegate = uiDelegate
    ...
}
```

example 3

```swift
@MainActor
private func makeView() -> Remakeable<EnhancedWKWebView> {
    let webView = Remakeable<EnhancedWKWebView> {
        let wrappedView = EnhancedWKWebView(frame: .zero, configuration: parent.configuration)
        parent.applyModifiers(to: wrappedView)
        return wrappedView
    }
    setUpWebViewProxy(webView)
    parent.applyModifiers(to: webView.wrappedValue)
    parent.loadInitialRequest(in: webView.wrappedValue)
    return webView
}
```

Reply:
fix in [310870a](https://github.com/cybozu/WebUI/pull/58/commits/310870aac85ae6c59f9d6f4fb58b490e2bfb03e9)